docs/content/getting-started/code-generation.md

```diff
@@ -79,29 +89,27 @@ $ cat data.yaml | hof gen yaml: - schema.cue -T template.txt
 
 ### Writing to file
 
-Use  `=` (equal) after the template name to write to file.
+The equal sign (`=`) is used to write to a file after the template name.
```

Comment:
this is a little confusing, I read this after as a temporal relation, but we want to communicate spacial.

i.e. the file name comes after the equal sign